WebRoot/css/zm.css

```diff
@@ -7139,6 +7139,27 @@ a.ZmLinkDisabled {
 	@ContactListPaginationLinkIcon@
 }
 
+/* Quick add contact dialog*/
+.ZmContactQuickAddDialog {
+	@ZmQuickAddDialog@
```

Comment:
For me the property name seem something generic for anything that's like QuickAdd. Contact/Appointment etc ...That's why I was looking to add something specific.